deployment/kubernetes/wave1.yaml.envsubst

```diff
@@ -53,6 +53,18 @@ items:
                   value: razee-io/mustachetemplate
                 - name: USER_AGENT_VERSION
                   value: "${WAVE1_VERSION}"
+                - name: MTP_RECONCILE_BY_DEFAULT
+                  valueFrom:
+                    configMapKeyRef:
+                      name: razeedeploy-overrides
+                      key: MTP_RECONCILE_BY_DEFAULT
+                      optional: true
+                - name: RR_RECONCILE_BY_DEFAULT
```

Comment:
this setting isn't in wave0 yet. it should be added there first

Reply:
https://github.com/answerbook/MustacheTemplate/pull/29